library/psa_crypto.c

```diff
@@ -3601,8 +3603,9 @@ psa_status_t psa_verify_hash_complete(
     status = psa_driver_wrapper_verify_hash_complete(operation);
 
     /* Update ops count with work done. */
-    operation->num_ops = psa_driver_wrapper_verify_hash_get_num_ops(
-        operation);
+    if (status == PSA_SUCCESS) {
```

Comment:
status could potentially be PSA_OPERATION_INCOMPLETE here as well as success. We should likely do this call regardless of status at this point, but only overwrite status if psa_driver_wrapper_verify_hash_get_num_ops() fails.